lib/configs/recommended.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
     'root/prefer-root-text': 'error',
     'root/preceded-by-await': ['error', { 'functionNames': ['pollForCondition', 'wait', 'waitForElement'] }],
     'root/prevent-async-test-without-it-slowly': 'error',
+    'root/prevent-non-smart-quotes-in-english-text': 'error',
```

Comment:
I feel like this rule shouldn't be in the recommended config since it's so tightly coupled to how the localization in root-mobile works. At some point could we come back around and remove all the root-mobile specific rules from here and put them directly in the root-mobile config?